src/hmacvalidator/hmacvalidator.go

```diff
@@ -0,0 +1,87 @@
+package hmacvalidator
```

Comment:
shall we call it utils/hmacvalidator.go ?

Reply:
you can, but package will still be `hmacvalidator` only the import path will change